packages/cli/src/utils/init.js

```diff
@@ -28,6 +30,70 @@ const initApp = async (path, createFunc) => {
   await removeDir(tempAppDir);
 };
 
+const testAuth = authType => {
+  // todo: oauth1 trello url?
+  const testUrl =
+    authType === 'digest'
+      ? 'https://httpbin.org/digest-auth/auth/myuser/mypass'
+      : 'https://auth-json-server.zapier-staging.com/me';
+
+  return `
+      const test = (z, bundle) => {
+        return z.request({
+          url: '${testUrl}',
+          ${authType === 'oauth2' ? inspect({}) : ''}
```

Comment:
I'm not sure what it is for. Is it intended or just a WIP thing?

Reply:
yep! that lets us print objects as strings